src/resources/views/components/mail/grid.blade.php

```diff
@@ -8,21 +8,21 @@
                 <td align="center" valign="top">
                     <table {{ $attributes->merge(['class' => "container grid grid-col-$columns"]) }} width="600" cellpadding="0" cellspacing="0" border="0">
                         <tr>
-                            <td class="mobile" align="center" valign="top">
+                            <td class="mobile" align="center" valign="top" {{ $one?->attributes }}>
                                 {{ $one ?: $slot }}
                             </td>
                             @if ($columns >= 2)
-                                <td class="mobile" align="center" valign="top">
+                                <td class="mobile" align="center" valign="top" {{ $two->attributes }}>
                                     {{ $two }}
                                 </td>
                             @endif
                             @if ($columns >= 3)
-                                <td class="mobile" align="center" valign="top">
+                                <td class="mobile" align="center" valign="top" {{ $three->attributes }}>
                                     {{ $three }}
                                 </td>
                             @endif
                             @if ($columns >= 4)
-                                <td class="mobile" align="center" valign="top">
+                                <td class="mobile" align="center" valign="top" {{ $four->attributes }}>
```

Comment:
This wont work as the component will simply use the default class, which in this case is 'mobile'. In order for this to work, you would need to merge the default class with whatever is passed when calling a component:

```blade
<td {{ $four->attributes->merge(['class' => 'mobile']) }} align="center" valign="top" >
    {{ $four }}
</td>
```

Reply:
@Korko Sorry for the late response, I was away. First of all, thank you for taking an intereset and your time!

I like the idea, however implementation needs to be tweaked a little bit in order for it to work. I left some comments/reviews detailing my take on the issue.